### PR TITLE
delete_arrow

### PR DESCRIPTION
**Pull request by ghost**
#### Changelog :

- Disable scrollup arrow.

### CODE REVIEW

File: src/javascript/app/base/page.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const Page = (() => {
                     // no default
                 }
             });
-            scrollToTop();
+            // scrollToTop();
```
Comment (negarn) [inline src/javascript/app/base/page.js]: IMO there's no need to comment the JS files. just commenting the HTML element is enough.

File: src/javascript/app/base/footer.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const Footer = (() => {
     };
 
     // by default elevio is 8px above bottom of page, and scrollup is 18px above elevio
-    const adjustElevioAndScrollup = (elevio_height = 8, scrollup_height = 18) => {
+    const adjustElevioAndScrollup = (elevio_height = 8,  scrollup_height = 18) => {
```
Comment (negarn) [inline src/javascript/app/base/footer.js]: extra space

Comment (negarn) [reply src/javascript/app/base/footer.js]: is it intended?

Comment (ghost) [reply src/javascript/app/base/footer.js]: @negarn nope

**Review by negarn — APPROVED** [review]

**Pull request merged**